packages/quick_blue/example/lib/main.dart

```diff
@@ -25,6 +25,7 @@ class _MyAppState extends State<MyApp> {
   @override
   void initState() {
     super.initState();
+    QuickBlue.setAvailabilityHandler(_handleAvailabilityChange);
```

Comment:
We'd better call `setAvailabilityHandler` only on Darwin platform until we implemeted others

`if (Platform.isIOS && Platform.isMacOS)`